app/graphql/mutations/create_need.rb

```diff
@@ -13,7 +13,7 @@ class Mutations::CreateNeed < Mutations::BaseMutation
   field :need, Types::NeedType, null: true
   field :errors, [String], null: false
 
-  def resolve(title:, point_of_contact:, description:, start_time:, end_time:, street_address:, city:, state:, zip_code:, supporters_needed:)
+  def resolve(title:, point_of_contact:, description:, start_time:, end_time:, street_address: nil, city: nil, state: nil, zip_code:, supporters_needed:)
```

Comment:
Sure thing! In testing that these fields were not required, I realized we were hitting an error because there was no default value in this method. I added the `nil` for the optional fields so that, if no value is entered, they are set to `nil`.